tests/integration/expected_output.md5

```diff
@@ -20,3 +20,5 @@ fe56b8577bada06fea45bb8835e7d7ee ./tests/integration/output/motevo/motif_HNRNPF_
 faa2f9e3e8f432575e281878d93a85a6 ./tests/integration/output/motevo/motif_HNRNPF_824/posteriors
 c0a4a9155ec7958b00bd3140e97c6dc9 ./tests/integration/output/motevo/motif_HNRNPF_823/posterior_sites
 20e7921795665bbe24e585e741705634 ./tests/integration/output/motevo/motif_HNRNPF_823/posteriors
+bb5480584d2aba7c8cfae0d5629deba9 ./tests/integration/output/combined_MotEvo_results.tsv
+0f62e3b0dc5bc8b048a021632ace92fe ./tests/integration/output/combined_MotEvo_results.csv
```

Comment:
There is also a corresponding `expected_output.txt` with the list of files, you should update that one as well.